src/elements/simple-input/src/index.tsx

```diff
@@ -52,7 +53,6 @@ export const SimpleInput = React.forwardRef(
           defaultValue={defaultValue}
           onFocus={() => setHasFocus(true)}
           onBlur={() => setHasFocus(false)}
-          {...props}
```

Comment:
Are you certain nothing was using this on the other forms?

Reply:
everything that was being spread from props that was required for the input were passed as indvidual props (placeholder and defaultValue) the remaining props that were being spread were not valid HTML attributes

Reply:
![image](https://user-images.githubusercontent.com/19416909/92230518-2cfbc980-eea3-11ea-8820-33e91321e629.png)
you can see here due to props being spread there are invalid html attributes (hasError and errors) due to them being spread in, I have tested locally on multistep form and the car insurance form and appears to be working correctly